src/app/layout.tsx

```diff
@@ -15,14 +14,13 @@ export const metadata: Metadata = {
 
 export default async function RootLayout({ children }: { children: React.ReactNode }) {
   const currentUser = getCurrentUser()
-  const apiClient: Planship = getPlanshipOnServer()
-  const entitlements = await apiClient.getEntitlements(currentUser.email)
-  const subscriptions = await apiClient.listSubscriptions(currentUser.email)
+  const subscriptions = await fetchSubscriptions(currentUser.email)
```

Comment:
This wrapping of the API and subsequent app-specific logic sorta detracts from the objective of providing an example app someone might use as a real world how-to-use-Planship. I don't have a good alternative besides adding comment blocks or handling the logic somewhere else with comments stating that you wouldn't need to do this for a real world app, hopefully leaving this code as it was.

Reply:
Elaborate on why you wouldn't need this in the real world app? If you rely on Planship for subscription management, the "real code" doesn't look very different from this, if the goal is to rended both subscription and entitlements info.

Reply:
The comments added to the functions called from here alleviate my concerns.